### PR TITLE
Redirects from login page and register page if user is logged in

### LINKED ISSUE #2936: Redirect from login page if user is logged in
Currently the ` accounts.login` view will still render even if there's already user logged in.

Since it doesn't make much sense for a user to log in twice, this view should check if there is a logged in user, and redirect to the post-login view if there is. (See #2882 for which view this should be).

---

**Good First Issue**: This issue is good for first time contributors. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by waseem18**
Resolves #2936

### CODE REVIEW

**Comment by waseem18** [comment]
@di  Could you please help me in reviewing this?

File: warehouse/accounts/views.py [hunk 1]
```diff
@@ -104,6 +104,8 @@ def login(request, redirect_field_name=REDIRECT_FIELD_NAME,
     # TODO: Logging in should reset request.user
     # TODO: Configure the login view as the default view for not having
     #       permission to view something.
+    if request.authenticated_userid is not None:
+        return HTTPSeeOther(request.route_path('manage.projects'))
```
Comment (pradyunsg) [inline warehouse/accounts/views.py]: This newly introduced branch doesn't seem to have a test for it.

Comment (waseem18) [reply warehouse/accounts/views.py]: Thanks for pointing out @pradyunsg 

Will address.

**Comment by di** [comment]
Thanks @waseem18!

**Pull request merged**